docs/patterns/clustered-listening-tentacles.md

```diff
@@ -0,0 +1,144 @@
+:::
+
+To avoid any issues with handling private keys you must specify an alternate location for the **tentacle.config** file upon installation of the Tentacle service. Follow these instructions to install Octopus Tentacle in a failover cluster using shared storage.
+
+## Requirements {#ClusteringTentacles-Requirements}
+
+This guide assumes you already have the following setup:
+
+- An Active Directory Domain and a local DNS server.
+- An Octopus Server (this does not need to be joined to the domain).
+- A two node active/passive Windows cluster where each node is joined to the domain.
+- A local IP Address available for the cluster.
+- Shared Storage configured for the cluster (in this example we are using E:\)
+
+## Installation {#ClusteringTentacles-Installation}
+
+:::warning
+**Installing Tentacles with Shared storage**
+This guide implements shared storage using an iSCSI target with Multipath IO confgured on the Tentacle servers, in this scenario it is best to avoid accessing the same iSCSI volume from two different hosts at the same time as doing so may resuilt in corrupt data on the iSCSI volume.
+:::
+
+On the first node, check that the shared drive is mounted and note down the Drive letter.
+
+*![](/docs/images/clustered-listening-tentacles/shared-disk-properties.jpg)*
+
+Run through the Tentacle MSI Installer to install Tentacle Manager to its default location (C:\Program Files\Octopus Deploy\Tentacle). Do not click "get started" in the Tentacle manager, install the Octopus Tentacle instance using the command prompt as an Administrator by opening `cmd` then run these commands (replacing relevant values as appropriate):
+
+```batchfile
+cd "C:\Program Files\Octopus Deploy\Tentacle\"
+Tentacle.exe create-instance --instance "Tentacle" --config "C:\Octopus\Tentacle.config"
+Tentacle.exe configure --instance "Tentacle" --app "<sharedStorageDriveLetter>:\Octopus\Applications" --home "<sharedStorageDriveLetter>:\Octopus\Home" --port "10933" --noListen "False"
+Tentacle.exe configure --instance "Tentacle" --trust "<OCTOPUS SERVER THUMBPRINT>"
+"netsh" advfirewall firewall add rule "name=Octopus Deploy Tentacle" dir=in action=allow protocol=TCP localport=10933
+Tentacle.exe service --instance "Tentacle" --install --stop --start
+```
+In the script, we:
+ - Installed the tentacle instance using the default instance name of "Tentacle" and made sure the Tentacle.config file was installed into the default location of "C:\Octopus\Tentacle.config". Since C:\ is local storage this file does not get swapped when we perform a swap to a new active node.
+ - Configured the new instance to listen on TCP Port 10933 and set the Application and Home directories to our Shared Storage drive.
+ - Configured the tentacle to trust our Octopus Server holding a certificate which matches the Certificate Thumbprint
```

Comment:
`which matches the Certificate Thumbprint` => `which matches the specified certificate thumbprint`